code/game/objects/items/crayons.dm

```diff
@@ -147,6 +147,29 @@
 	colour = "#DA00FF"
 	colourName = "purple"
 
+/obj/item/toy/crayon/random/New()
+	icon_state = pick(list("crayonred", "crayonorange", "crayonyellow", "crayongreen", "crayonblue", "crayonpurple"))
+	switch(icon_state)

```

Comment:
Why no white in here? You crayon racist.

Semi-related: we don't have black crayons, do we? Totally not intending the race card here, but I _could_ :^)

Off-topic: What would be the opinion of RGB crayons? Maybe random a lot more interesting, but then you'd need a method of getting any color you want. Botany bees mabee?